Public/Src/Engine/Processes/FileAccessManifest.cs

```diff
@@ -785,6 +836,7 @@ public ArraySegment<byte> GetPayloadBytes(FileAccessSetup setup, MemoryStream st
             {
                 WriteDebugFlagBlock(writer, ref debugFlagsMatch);
                 WriteInjectionTimeoutBlock(writer, timeoutMins);
+                WriteChildProcessesToBreakAwayFromSandbox(writer, ChildProcessesToBreakawayFromSandbox);

```

Comment:
you may want to do this conditionally `#if PLATFORM_WIN`; otherwise you'll have to fix the FAM parser for Mac

Reply:
So if I'm getting this right, I think I fixed the manifest parser already on the mac side (to just ignore the list for now)

Reply:
yes, sorry, I hadn't reached that part yet when I wrote this, and from the description I assumed you hadn't changed anything on the Mac side